msteams-platform/m365-apps/combine-office-add-in-and-teams-app.md

```diff
@@ -0,0 +1,621 @@
+
+Combining an Outlook Add-in with a Teams app enables scenarios that can't be satisfied with either of them separately. For example, consider an app that enables a salesperson to insert a customized discount in an email to a customer and records the offer in a backend database. The sales manager can also view data about all the discounts that have been offered in a Teams tab. For more information, see [Discount Offers sample](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/tab-add-in-combined/nodejs).
+
+This article will walk you through the steps to add an Outlook Add-in to a Teams app, test it, then deploy it to Azure.
+
+## Prerequisites 
+
+To configure an Office Add-in, ensure that you meet the following requirements: 
+
+- A Microsoft 365 account that includes Teams to test the application. Alternatively, you can have separate subscriptions to both Microsoft 365 and Teams. For example, a test account with *.onmicrosoft.com through the [Microsoft 365 Developer Program](/office/developer-program/microsoft-365-developer-program). 
+- Your Microsoft 365 account is added as an account in desktop Outlook. For more information, see [add an email account to Outlook](https://support.microsoft.com/office/add-an-email-account-to-outlook-e9da47c4-9b89-4b49-b945-a204aeea6726).
+ - An Azure account with active subscription to deploy the Teams app to Azure. If you don't already have one, you can create your [free Azure account](https://azure.microsoft.com/free/).
+ - A Teams app created using the latest version of Teams Toolkit.
+
+## Overview
+
+The following are the major steps to adding an Outlook Add-in to a Teams app.
+
+1. [Prepare the Teams app project](#prepare-the-teams-app-project)
+1. [Create an Office Add-in project](#create-an-outlook-add-in-project) that is initially separate from your Teams app project.
+1. [Merge the manifest](#merge-the-manifest) from the Outlook Add-in project into the unified manifest for Microsoft 365.
+1. [Copy the Outlook Add-in files to the Teams app project](#copy-the-outlook-add-in-files-to-the-teams-app-project).
+1. [Edit the tooling configuration files](#edit-the-tooling-configuration-files).
+1. [Run the app and add-in locally at the same time](#run-the-app-and-add-in-locally-at-the-same-time)
+1. [Move the application to Azure](#move-the-application-to-azure).
+
+## Prepare the Teams app project
+
+Begin by separating the source code for the tab (or bot) into its own subfolder. These instructions assume that the project initially has the following structure. To create a Teams app project with this structure, be sure you're using the latest released version of Teams Toolkit. 
+
+```
```

Comment:
We add the structure as image and not as code snippet. Please check if it can be added as image.

Reply:
> We add the structure as image and not as code snippet. Please check if it can be added as image.

I understand, but this structure is likely to change frequently when there are new versions of Teams Toolkit. So, to make it easy to update, it should be an editable snippet.

Reply:
Agreed. Thanks.